src/cli/upload/stdin-to-uss-file/StdinToUssFile.Handler.ts

```diff
@@ -18,19 +18,25 @@ export default class UploadStdinToUssFileHandler extends FTPBaseHandler {
     public async processFTP(params: IFTPHandlerParams): Promise<void> {
 
         const ussFile = UssUtils.normalizeUnixPath(params.arguments.ussFile);
-        const content: Buffer | string = await CoreUtils.readStdin(params.stdin);
-
-        const options = {
-            content,
-            transferType: params.arguments.binary ? TRANSFER_TYPE_BINARY : TRANSFER_TYPE_ASCII,
-        };
-        await UssUtils.uploadFile(params.connection, ussFile, options);
-
-        const uploadSource: string = "stdin";
-        const successMsg = params.response.console.log("Uploaded from %s to %s ", uploadSource, ussFile);
-        params.response.data.setMessage(successMsg);
-        this.log.info(successMsg);
-
+        if (!ussFile.startsWith('/')) {
```

Comment:
Good to have this check in `UssUtils.checkAbsoluteFilePath(filePath)`, which returns error message when the check fails. In this way, we can avoid having the check and error message duplicate in two places.

Reply:
Thank you @std4lqi ! I add checkAbsoluteFilePath in UssUtils and add test in UssUtils.test.ts.